test/index.test.js

```diff
@@ -1,9 +0,0 @@
-'use strict';
-
-const assert = require('chai').assert;
-
-describe('index test', () => {
```

Comment:
@petey We're not sure how to test this actually. The `perform` function gets called when something is enqueued (which is not in the code). That means we need to mock the queue, and basically everything, essentially testing nothing.